designer/server/src/common/templates/layouts/page.njk

```diff
@@ -12,7 +12,13 @@
   {% include "partials/navigation.njk" %}
 {% endblock %}
 
-{% block pageTitle %}{{ pageTitle }} - {{ config.serviceName }}{% endblock %}
+{% if errorList | length %}
+  {% set errorPrefix = "Error: " %}
+{% else %}
+  {% set errorPrefix = "" %}
+{% endif %}
+
+{% block pageTitle %}{{ errorPrefix }}{{ pageTitle }} - {{ config.serviceName }}{% endblock %}
```

Comment:
Good idea!

It's why I had to use `pageHeading or pageTitle` to return a non-appending header!

Only problem now is the create form journey 😣

I've copied CDP and used `formErrors` _not_ `errorList` which means `pageTitle` still needs appending on those pages. Plus they use `buildErrorDetails()` which returns a slightly incompatible array

Probably best keeping the old way for now, until we can consolidate?

```njk
{% if formErrors | length %}
  {% set pageTitle = "Error: " + pageTitle %}
{% endif %}
```

Reply:
Top Nunjucks trick though if you want to slim this down

**Trimmed**
```
{%- -%}
{{- -}}
```

**Untrimmed**
```
{% %}
{{ }}
```

```suggestion
{% block pageTitle -%}
  {{ "Error: " if errorList | length }}{{ pageTitle }} - {{ config.serviceName }}
{%- endblock %}
```

Reply:
Weirdly enough I tried exactly that and it broke with some super weird error, hence my slightly verbose result.

Reply:
I'll move this bit out to a separate PR branch/PR.